plaidml/bridge/openvino/tests/functional/shared_tests_instances/single_layer_tests/activation.cpp

```diff
@@ -63,23 +75,49 @@ std::map<std::vector<size_t>, std::vector<std::vector<size_t>>> preluBasic = {
-                                           ::testing::Values(CommonTestUtils::DEVICE_PLAIDML)                     //
+const auto basicCases = ::testing::Combine(                                //
+    ::testing::ValuesIn(CommonTestUtils::combineParams(activationTypes)),  //
+    ::testing::ValuesIn(netPrecisions),                                    //
+    ::testing::Values(InferenceEngine::Precision::UNSPECIFIED),            //
+    ::testing::Values(InferenceEngine::Precision::UNSPECIFIED),            //
+    ::testing::Values(InferenceEngine::Layout::ANY),                       //
+    ::testing::Values(InferenceEngine::Layout::ANY),                       //
+    ::testing::ValuesIn(CommonTestUtils::combineParams(basic)),            //
+    ::testing::Values(CommonTestUtils::DEVICE_PLAIDML)                     //
 );
 
-const auto basicPreluCases =
-    ::testing::Combine(::testing::ValuesIn(CommonTestUtils::combineParams(activationParamTypes)),  //
-                       ::testing::ValuesIn(netPrecisions),                                         //
-                       ::testing::ValuesIn(CommonTestUtils::combineParams(preluBasic)),            //
-                       ::testing::Values(CommonTestUtils::DEVICE_PLAIDML));
+const auto basicPreluCases = ::testing::Combine(                                //
+    ::testing::ValuesIn(CommonTestUtils::combineParams(activationParamTypes)),  //
+    ::testing::ValuesIn(netPrecisions),                                         //
+    ::testing::Values(InferenceEngine::Precision::UNSPECIFIED),                 //
+    ::testing::Values(InferenceEngine::Precision::UNSPECIFIED),                 //
+    ::testing::Values(InferenceEngine::Layout::ANY),                            //
+    ::testing::Values(InferenceEngine::Layout::ANY),                            //
+    ::testing::ValuesIn(CommonTestUtils::combineParams(preluBasic)),            //
+    ::testing::Values(CommonTestUtils::DEVICE_PLAIDML)                          //
+);
 
-INSTANTIATE_TEST_CASE_P(smoke, ActivationLayerTest, basicCases, ActivationLayerTest::getTestCaseName);
-INSTANTIATE_TEST_CASE_P(smoke_Prelu, ActivationLayerTest, basicPreluCases, ActivationLayerTest::getTestCaseName);
+INSTANTIATE_TEST_CASE_P(                  //
+    smoke_Activation_Basic,               //
+    ActivationLayerTest,                  //
+    basicCases,                           //
+    ActivationLayerTest::getTestCaseName  //
+);
 
-// TODO: Fix & re-enable once no longer broken for CPU plugin as well
-// INSTANTIATE_TEST_CASE_P(smoke_Prelu_Param, ActivationParamLayerTest, basicPreluCases,
-//                         ActivationLayerTest::getTestCaseName);
+INSTANTIATE_TEST_CASE_P(                  //
+    smoke_Activation_Basic_Prelu,         //
+    ActivationLayerTest,                  //
+    basicPreluCases,                      //
+    ActivationLayerTest::getTestCaseName  //
+);
+
+// ActivationParamLayerTest fails in nGraph
```

Comment:
Is this also commented out in CPU plugin version?

Reply:
No, this is not commented in CPU plugin. However it fails when generating expected output from reference implementation. We will furthur check this issue.

Reply:
Ok, sounds good. Odd that the reference implementation fails here but not with the CPU plugin.